doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -7,7 +7,7 @@
 ## Table of Contents
 * [2.1 Introduction.](#2.1)
 * [2.2 Reference Model Requirements.](#2.2)
-* [2.3 Kubernetes Requirements.](#2.3)
+* [2.3 Containerised Platform Requirements.](#2.3)
```

Comment:
I thought we dropped the "platform" to avoid confusion, and went with "CaaS"?

Reply:
I would suggest this is set to "Kubernetes Architecture Requirements", as this is what we're doing. I agree on not using the term "Platform" as it is too contentious.

Reply:
Thanks -- change made